src/EditorFeatures/CSharpTest/Diagnostics/AddExplicitCast/AddExplicitCastTests.cs

```diff
@@ -1049,32 +1049,6 @@ public async Task InheritInterfaces5()
             await TestInRegularAndScriptAsync(
             @"
 class Program
-{
-    interface Base1 {}
-    interface Base2 : Base1 {}
-
-    void Foo(Base1 b) {
-        Base2 b2 = [||]b;
-    }
-}",
-            @"
-class Program
-{
-    interface Base1 {}
-    interface Base2 : Base1 {}
-
-    void Foo(Base1 b) {
-        Base2 b2 = (Base2)b;
-    }
-}");
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddExplicitCast)]
-        public async Task InheritInterfaces6()
```

Comment:
why was this test removed?

Reply:
It is a same test with the previous one, so I removed it